docs/resources/Guild.md

```diff
@@ -774,13 +775,14 @@ Modify attributes of a [guild member](#DOCS_RESOURCES_GUILD/guild-member-object)
 
 ###### JSON Params
 
-| Field      | Type                | Description                                                                                            | Permission       |
-| ---------- | ------------------- | ------------------------------------------------------------------------------------------------------ | ---------------- |
-| nick       | string              | value to set users nickname to                                                                         | MANAGE_NICKNAMES |
-| roles      | array of snowflakes | array of role ids the member is assigned                                                               | MANAGE_ROLES     |
-| mute       | boolean             | whether the user is muted in voice channels. Will throw a 400 if the user is not in a voice channel    | MUTE_MEMBERS     |
-| deaf       | boolean             | whether the user is deafened in voice channels. Will throw a 400 if the user is not in a voice channel | DEAFEN_MEMBERS   |
-| channel_id | snowflake           | id of channel to move user to (if they are connected to voice)                                         | MOVE_MEMBERS     |
+| Field                        | Type                | Description                                                                                            | Permission       |
+| ---------------------------- | ------------------- | ------------------------------------------------------------------------------------------------------ | ---------------- |
+| nick                         | string              | value to set users nickname to                                                                         | MANAGE_NICKNAMES |
+| roles                        | array of snowflakes | array of role ids the member is assigned                                                               | MANAGE_ROLES     |
+| mute                         | boolean             | whether the user is muted in voice channels. Will throw a 400 if the user is not in a voice channel    | MUTE_MEMBERS     |
+| deaf                         | boolean             | whether the user is deafened in voice channels. Will throw a 400 if the user is not in a voice channel | DEAFEN_MEMBERS   |
+| channel_id                   | snowflake           | id of channel to move user to (if they are connected to voice)                                         | MOVE_MEMBERS     |
+| communication_disabled_until | ?ISO8601 timestamp  | how long the user can not interact with the guild; set to null to remove timeout                       | KICK_MEMBERS     |
```

Comment:
api accepts end timestamps up to 28 days in the future

```suggestion
| Field                        | Type                | Description                                                                                                                | Permission       |
| ---------------------------- | ------------------- | -------------------------------------------------------------------------------------------------------------------------- | ---------------- |
| nick                         | string              | value to set users nickname to                                                                                             | MANAGE_NICKNAMES |
| roles                        | array of snowflakes | array of role ids the member is assigned                                                                                   | MANAGE_ROLES     |
| mute                         | boolean             | whether the user is muted in voice channels. Will throw a 400 if the user is not in a voice channel                        | MUTE_MEMBERS     |
| deaf                         | boolean             | whether the user is deafened in voice channels. Will throw a 400 if the user is not in a voice channel                     | DEAFEN_MEMBERS   |
| channel_id                   | snowflake           | id of channel to move user to (if they are connected to voice)                                                             | MOVE_MEMBERS     |
| communication_disabled_until | ?ISO8601 timestamp  | when the user should be able to interact with the guild again (up to 28 days in the future); set to null to remove timeout | KICK_MEMBERS     |
```

Reply:
Really? It would be nice if they don't set any limits. If you don't mind, can I ask the source of that please?

Reply:
@Raiden-Ei You will get more informations on this soon enough.
This is not released.

Reply:
> can I ask the source of that please

getting the error and then trying timestamps until i found the limit